.changeset/tall-clouds-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Added --persist and --persist-to in place of --experimental-enable-local-persistence. Changed the default persistence directory to .wrangler/state, relative to wrangler.toml.
```

Comment:
Perhaps make is more clear that the `--experimental-...` flag is no longer valid for those scanning the changelog?
Also, perhaps provide steps for renaming the previous storage path to the new default?